ts/features/profile/store/reducers/__tests__/profile.test.ts

```diff
@@ -0,0 +1,10 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { appReducer } from "../../../../../store/reducers";
+import { applicationChangeState } from "../../../../../store/actions/application";
+
+describe("Test profile reducer behaviour", () => {
+  it("The initial state should be pot.none", () => {
+    const globalState = appReducer(undefined, applicationChangeState("active"));
+    expect(globalState.features.profile).toEqual(pot.none);
+  });
```

Comment:
Why you didn't add the other tests? Can be useful

Reply:
Sure but I have in this PR a partially reducers yet... I will add more tests next! 😃